package.json

```diff
@@ -65,14 +66,13 @@
     "enzyme": "2.9.1",
     "eslint": "3.19.0",
     "eslint-config-wordpress": "1.1.0",
-    "eslint-plugin-jest": "20.0.3",
+    "eslint-plugin-jest": "21.2.0",
     "eslint-plugin-jsx-a11y": "4.0.0",
     "eslint-plugin-react": "6.10.3",
     "expose-loader": "0.7.3",
     "extract-text-webpack-plugin": "3.0.0",
     "gettext-parser": "1.3.0",
-    "jest": "20.0.4",
-    "jest-junit-reporter": "1.1.0",
```

Comment:
We don't use it at all.

Reply:
In my testing, it was being required as an enzyme peer dependency

Reply:
That's strange, I think I added it because we have that in Calypso to integrate with CircleCI.

Reply:
Yes, it was me with https://github.com/WordPress/gutenberg/commit/1406fe8b6548ea5d516927d3e22553e6bf60e9dc.

Reply:
Yes, but when upgrading to enzyme3, I dropped it and I had to add it again because it was a peer dependency. Maybe not with the current version of enzyme though